form/root_storage/demangle_name.cpp

```
@@ -7,27 +7,28 @@
-    operator char const*() const { return m_cstr.get(); } // NOLINT(google-explicit-constructor)
+    operator char const*() const { return cstr_.get(); } // NOLINT(google-explicit-constructor)
   private:
-    std::unique_ptr<char, decltype(&std::free)> m_cstr;
+    std::unique_ptr<char, decltype(&std::free)> cstr_;
   };
 }
 
 namespace form::detail::experimental {
   // Return the demangled type name
-  std::string DemangleName(std::type_info const& type)
+  std::string demangle_name(std::type_info const& type)
```

Cppcheck (2.21.0)

[style] The function 'demangle_name' is never used.

(unusedFunction)